data/json/recipes/recipe_deconstruction.json

```diff
@@ -2436,5 +2436,20 @@
       [ [ "plastic_chunk", 10 ] ],
       [ [ "solder_wire", 5 ] ]
     ]
+  },
+  {
+    "result": "electric_blanket",
+    "type": "uncraft",
+    "skill_used": "tailor",
+    "time": 30000,
+    "difficulty": 1,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "rag", 25 ] ],
+      [ [ "element", 5 ] ],
+      [ [ "cable", 5 ] ],
+      [ [ "plastic_chunk", 10 ] ],
```

Comment:
you supposed to be able to get more plastic chunks from it than you put in? the recipe only takes 4

Reply:
Nope, that's a stray zero. Thanks for spotting that.